web/static/index.html

```diff
@@ -26,6 +26,7 @@
     <!-- main.js used as general js file for index.html -->
     <script src="/js/main.js"></script>
     <script src="/js/explore.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
@spiderxm : why do we need this ?

Reply:
I forgot to remove it. Previously I was trying to use font-awesome icon. Will remove it.

Reply:
removed